rke/k8s_version_info.go

```diff
@@ -40,16 +40,18 @@ func loadRancherDefaultK8sVersions() map[string]string {
 		"2.6.5":        "v1.23.x",
 		"2.6.6":        "v1.23.x",
 		"2.6.7":        "v1.24.x",
+		"2.7.0":        "v1.24.x",
+		"2.7.1":        "v1.24.x",
 		// rancher will use default if its version is absent
-		"default": "v1.24.x",
+		"default": "v1.25.x",
```

Comment:
This will break rancher 2.6.x versions > 2.6.7 because default is 1.25 now but 1.25 isn’t available for 2.6.x.

Reply:
as mentioned in the second point in the issue 
```
We do not need to add other 2.6.x versions as this branch is used for v2.7.x versions only. For the same reason we're not doing any updates for k8s_version_info.go in dev-v2.6 KDM branch as 1.24.x stays default for 2.6.x releases.
```
i think it should be fine since this branch is going to be used only for 2.7 releases.

Reply:
Thanks, this makes sense.